docs/operators/rest.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "@rest"
```

Comment:
The guide and operator are two separate documents. I think we should keep it that way. What you have written is more of a guide than the operator documentation.